yard.gemspec

```diff
@@ -21,4 +21,7 @@ Gem::Specification.new do |s|
   s.executables   = ['yard', 'yardoc', 'yri']
   s.license = 'MIT' if s.respond_to?(:license=)
   s.metadata['yard.run'] = 'yri'
+
+  # FIXME: The following dependency can be removed when dropping legacy Ruby support.
+  s.add_runtime_dependency('e2mmap')
```

Comment:
e2mmap isn't needed, the library isn't actually used anywhere in the file. We can remove the require call instead. Tests still pass without it for me.